test/winter_onboarding_2021/fleet_management_service/handlers/cab_test.clj

```diff
@@ -16,10 +16,10 @@
 
 (deftest add-cab
   (testing "POST /cabs/ endpoint with valid cab data, shoudld redirect to '/cabs/<<uuid of new cab>> "
-    (let [response (handlers/create {:multipart-params
-                                     {:name "Test cab"
-                                      :licence-plate "KA20X1234"
-                                      :distance-travelled "1223"}})
+    (let [response (handlers/create {:form-params
+                                     {:cabs/name "Test cab"
+                                      :cabs/licence-plate "KA20X1234"
+                                      :cabs/distance-travelled "1223"}})
```

Comment:
I am not sure this is the correct approach here.
Because handlers do not receive namespaced keys from the client, our handler code should also not expect this.

Reply:
Yes, I have changed the same